python/cuml/manifold/t_sne.pyx

```diff
@@ -485,3 +485,8 @@ class TSNE(Base):
             "pre_momentum",
             "post_momentum",
         ]
+
+    def _more_tags(self):
+        return {
+            'preferred_input_order': 'C'
```

Comment:
Just leaving a small note (more for myself) that t_sne & UMAP both could probably accept 'F' now that the underlying KNN prim can accept it.